### PR TITLE
Adding notes around "how to contribute"

### PR DESCRIPTION
**Pull request by nisheetjain**
@subhagpo checkout

### CODE REVIEW

**Comment by MSLaguana** [comment]
Looks good to me

File: README.md [hunk 1]
```diff
@@ -61,6 +61,61 @@ taco help
...
+```
+gulp
+```
+Above command will build and install TACO packages under TACO\build\packages\node_modules.
+
+### Running TACO
+Once TACO has been built, it can be run from built location.
+* To run taco-cli run 
+```
+build/packages/node_modules/taco-cli/bin/taco
```
Comment (dlebu) [inline README.md]: Wouldn't it be easier to advice users to `npm link` the package? That way, every time they make a change and compile, they can run `taco` directly without specifying the path.

File: README.md [hunk 2]
```diff
@@ -61,6 +61,61 @@ taco help
...
+
+### Getting tests coverage
+
+To check test coverage, rlease run following command
+```
+gulp coverage
+```
+
+### Coding guidelines
+TACO uses tslint rules specified in [tslint.json] (https://github.com/Microsoft/TACO/blob/master/tools/tslint.json). Run following command to make sure code is tslint clean
```
Comment (dlebu) [inline README.md]: Do we want to add guidelines for code quality as well? I am thinking about code reuse, modularity, extensibility, etc.

File: README.md [hunk 3]
```diff
@@ -61,6 +61,61 @@ taco help
...
+### Running tests
+
+Please run following to make sure all tests are passing
+```
+gulp run-tests
+```
+
+### Getting tests coverage
+
+To check test coverage, rlease run following command
```
Comment (meenakb) [inline README.md]: minor typo

File: README.md (hunk 1 above)
Comment (nisheetjain) [inline README.md]: I can try this if we can make this workflow easier

File: README.md (hunk 2 above)
Comment (meenakb) [inline README.md]: +1 to @dlebu suggests.

**Comment by dlebu** [comment]
LGTM!

**Pull request merged**